Gemfile.lock

```diff
@@ -823,4 +832,4 @@ RUBY VERSION
    ruby 3.0.3p157
 
 BUNDLED WITH
-   2.1.4
+   2.2.32
```

Comment:
I'm wondering if this change is necessary here?
Local setup:
```
➜  openfoodnetwork git:(master) bundle -v
Bundler version 2.1.4
```

Is this necessary to update the bundle version? Maybe. Is this intentional ?

Reply:
Well spotted, thanks :pray: 

That was unintentional - it seems that I was running v2.2.32 locally. I thought it would automatically downgrade by doing `bundle install` after pulling master... But it didn't so I've manually downgraded back to v2.1.4:

```
(base) ffurtado@ff-LAT:~/openfoodnetwork$ bundler --version
Bundler version 2.2.32
(base) ffurtado@ff-LAT:~/openfoodnetwork$ gem install bundler -v 2.1.4
Fetching bundler-2.1.4.gem
Successfully installed bundler-2.1.4
1 gem installed
(base) ffurtado@ff-LAT:~/openfoodnetwork$ bundler --version
Bundler version 2.1.4
```

 and repeated the adding of the debug gem, then force pushed. Should be ok now.